docs/userGuide/syntax/code.md

```diff
@@ -278,6 +278,13 @@ A _wrap text_ button can be added to code blocks using the `codeBlockWrapButtons
 </panel>
 <br>
 
+##### Printing optimization
+
+Markbind automatically enhances the readability of your code blocks when printing:
+- **Light theme for clarity**: Ensures optimal contrast by switching code blocks to the light theme when printing.
```

Comment:
Hi @Tim-Siu, where did you see this? Because the printing still seems to be in dark theme if the code is in dark theme
![image](https://github.com/MarkBind/markbind/assets/77204346/b0073e9d-2121-489c-9772-19f58be7f319)

Reply:
<img width="865" alt="Screenshot 2024-02-25 at 17 29 31" src="https://github.com/MarkBind/markbind/assets/61866948/f5c68787-be58-4795-96ae-2fd48ca4566b">
Hi @yucheng11122017 
It does work like that on my side. But indeed this seems to be undefined behaviors. I will look into this. I will set the PR to draft for now until I thoroughly understand the CSS for the site. Sorry for the overlook.

Reply:
Hi no worries @Tim-Siu! I think for now, this PR can just focus on adding the line numbers, which is what you already did. You can just clean up the documentation to not include these undefined behaviors first, add an issue, and then fix it in the future if you are interested!